src/Umbraco.Web.UI.Client/src/views/propertyeditors/multiurlpicker/multiurlpicker.controller.js

```diff
@@ -144,16 +144,6 @@ function multiUrlPickerController($scope, angularHelper, localizationService, en
         if ($scope.model.validation && $scope.model.validation.mandatory && !$scope.model.config.minNumber) {
             $scope.model.config.minNumber = 1;
         }
-
-        _.each($scope.model.value, function (item){
-            // we must reload the "document" link URLs to match the current editor culture
-            if (item.udi.indexOf("/document/") > 0) {
-                item.url = null;
-                entityResource.getUrlByUdi(item.udi).then(function (data) {
-                    item.url = data;
-                });
-            }
-        });
```

Comment:
The PR also contains the fix applied in PR https://github.com/umbraco/Umbraco-CMS/pull/7543, so the Multi URL Picker works again with external URLs.

Reply:
I've now applied the merged commit from PR https://github.com/umbraco/Umbraco-CMS/pull/7623.